src/version.cpp

```diff
@@ -166,7 +166,7 @@ static std::vector<std::string> getLoadedLibraries() {
   snprintf(procsz, sizeof(procsz), "/proc/%d/path/a.out", getpid());
   int l = readlink(procsz, pathsz, sizeof(pathsz));
   if (l > 0) {
-    pathsz[l] = '\0';
+    pathsz[l - 1] = '\0';
```

Comment:
Based on what I've read in the manpage for `readlink`, I think this is the correct way to do it:

```c
ssize_t l = readlink(procsz, pathsz, sizeof(pathsz) - 1);
if (l > 0) {
  pathsz[l] = '\0';
```

Reply:
Fixed.